app/registries/views.py

```diff
@@ -292,12 +293,22 @@ def get_queryset(self):
             else:
```

Comment:
Even though it won't make a practical difference (due to the "if user not staff" check on line 299), I think it would still be better to avoid the code on 291-292 from running if user not staff